Gemfile.lock

```diff
@@ -669,7 +669,7 @@ GEM
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       railties (>= 3.0)
-    warden (1.2.6)
+    warden (1.2.7)
```

Comment:
I wouldn't upgrade this so we can merge this PR without further ado

Reply:
I'm really reluctant to hack Gemfile.lock files manually. On the other hand, release notes for warden 1.2.7 mentioned an upgrade to Ruby 2.3.
I'll make the change, but I'm expressing my overall unhappiness with the situation (both the really old version of Ruby and having to hack the lock file)

Reply:
@sauloperez I replicate this when testing on my local machine, and I tend to agree with @andypalmer that hacking the lock file is something we should avoid. A quick look at the [ruby 2.3 PR](https://github.com/hassox/warden/pull/139) that is mentioned in the release notes suggests to me that it is designed to add compatibility for 2.3, rather than deprecate the use of older versions.